src/main/groovy/com/etendoerp/legacy/dependencies/container/EtendoCoreJarArtifact.groovy

```diff
@@ -45,13 +45,23 @@ class EtendoCoreJarArtifact extends ArtifactDependency{
             include "${JAR_ETENDO_LOCATION}"
         }
 
+        /*
+            If the system separator is "\" it must be replaced by "/"
+            to prevent the java.lang.String.replaceFirst method from
+            trying to escape characters in the file path when extracting, causing
+            java.util.regex.PatternSyntaxException
+        */
+        String metaInfPath = "\\" != File.separator ?
```

Comment:
Check SO.

Reply:
File.separator is checking for the system separator. I really dont care about the system, just the separator of it